scripts/operator.sh

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2021-present, StarRocks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+#! /bin/bash
+
+# set the home path
+HOME_PATH=$(
+  cd "$(dirname "$0")"
+  cd ..
+  pwd
+)
+
+cd "$HOME_PATH/helm-charts/charts/kube-starrocks/charts/operator"
+
+helm template -n starrocks . >/tmp/operator.yaml
+
+# create values.yaml for the parent chart
+cat <<EOF >$HOME_PATH/deploy/operator.yaml
+# This file is generated by scripts/operator.sh.
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: cn-controller-manager
```

Comment:
shall change this label? `starrocks-controller-manager` or just removed the labels.